internal/services/containers/kubernetes_flux_configuration_resource_test.go

```diff
@@ -843,3 +906,31 @@ resource "azurerm_kubernetes_flux_configuration" "test" {
 }
 `, template, data.RandomInteger)
 }
+
+func (r KubernetesFluxConfigurationResource) gitRepositoryProvider(data acceptance.TestData, providerType string) string {
+	template := r.template(data)
+	return fmt.Sprintf(`
+%s
```

Comment:
All of the other tests configuration helper functions `%s` declarations have odd indentation (e.g., "                                   %s") can you correct these as well to be at column position 0?